src/pages/[...blog].astro

```diff
@@ -23,9 +23,50 @@ window.addEventListener('DOMContentLoaded', () => {
   // When this script is executed for any blog posts that are written as .mdx format, the logic inside prepFootnotes() does not work for some reason.
   // But giving it a reasonable amount of delay ensures it consistently works for both .md and .mdx blog posts.
   // (reference: https://github.com/okTurtles/groupincome.org/pull/107#discussion_r1547356061)
-  setTimeout(prepFootnotes, 100)
+  setTimeout(() => {
+    prepFootnotes()
+
+    const url = new URL(window.location.href)
```

Comment:
While working on the #108 , realised that the `#` anchor scroll behaviour doesn't work upon the initial page load, so made a fix for it here.